src/parser.js

```diff
@@ -224,6 +224,8 @@ define([
 
         var mug = form.mugTypes.make(role, form);
         mug.p.nodeID = nodeID;
+        mug.__originalNodeID = nodeID;
```

Comment:
Should this use `nodeID.toLowerCase()`? It's possible (with hand-edited XML at least) that an XML element name may have uppercase letters, and I think that would make it impossible to resolve the info message.

Reply:
Hmm I see the inconsistency.  Would it be better to remove the ```.toLowerCase``` in ```validationFunc```?  I'm assuming question IDs are case sensitive based on my experiments with formbuilder.

Reply:
I actually don't know if question IDs are case sensitive—that would be a question for the mobile team. Also probably good to find that out since `toLowerCase` should not be used anywhere in this check if they are.

@ctsims or @shubham1g5 do you know the answer?

Reply:
@millerdev  I think so. I was able to create 2 question in a form "mother_name" and "mother_Name" and save different values for them.

Reply:
Hi Daniel,

It is technically the case that XML QNames are case sensitive, but I am absolutely supportive of Vellum lower-casing all xml identifiers if that's an approach we are already, as I think it's extremely unhelpful for our users to have to deal with that reality.